packages/app/src/navigation/KeyboardBindingsModal.spec.tsx

```diff
@@ -1,9 +1,16 @@
 import KeyboardBindingsModal from './KeyboardBindingsModal.vue'
+import { defaultMessages } from '@cy/i18n'
 
 describe('SidebarNavigationRow', () => {
```

Comment:
This one would be my fault...

```suggestion
describe('KeyboardBindingsModal', () => {
```